### DISCUSSION TITLE
manually load a checkpoint

### DISCUSSION THREAD

**Discussion by foldl** [google/flax#4280] [Q&A]
I am investigating how to load `state_dict` from a checkpoint file. For a `Dense` module with `bias = False`, I can see something like this:

```
hidden0:
     kernel:
         v_row:
             tensor (1024,)
         v_col:
             tensor (4096,)
         v:
             tensor (1,)
         m:
             tensor (1024, 4096)
```

`m` shall be the matrix itself. But how about other ones?

**Comment by foldl** [comment]
These are tensors generated in training stage.

**Comment by 8bitmp3** [comment]
Hi @foldl , just looping in @IvyZX
